src/interfaces/ISablierV2Lockup.sol

```diff
@@ -131,6 +132,11 @@ interface ISablierV2Lockup is
     /// @param streamId The stream id for the query.
     function isWarm(uint256 streamId) external view returns (bool result);
 
+    /// @notice Retrieves the maximum broker fee that can be charged by the broker, denoted as a fixed-point
+    /// number where 1e18 is 100%.
+    /// @dev This value is hard coded as a constant.
+    function MAX_BROKER_FEE() external view returns (UD60x18);
```

Comment:
Shouldn't this have remained in `SablierV2Base`?

Reply:
`SablierV2Base` no longer exists - keeping the contract only for one constant made no sense.

Reply:
ok